### DISCUSSION TITLE
minimize the amount of new syntax where possible

### DISCUSSION THREAD

**Discussion by kyunghoon** [carbon-language/carbon-lang#1629] [Language design]
I was wondering what the rationale was for the new instance method syntax `[me: Self]`. Was there an issue with just passing it in as the first argument, similar as to how it is done is rust? Less syntax to learn FTW.

Great project, c++ has needed a reboot for a very long time.

**Comment by geoffromer** [comment]
See [p0722](https://github.com/carbon-language/carbon-lang/blob/trunk/proposals/p0722.md) for the rationale behind this choice. Basically, we wanted the contents of the parentheses in a function call to match the contents of the parentheses in the declaration of the function.
